examples/browser-webrtc/client/README.md

```diff
@@ -0,0 +1,7 @@
+# Use rust-libp2p in the Browser
+
+## Run Leptos App
+
+Ensure you have `trunk` [installed](https://trunkrs.dev/).
+
+From this directory, run `trunk serve --open`
```

Comment:
I am not deeply familiar with the WASM Rust ecosystem. I would like to keep the required tools to be installed minimal. In our WebTransport tests we are already using `wasm-pack`. Would that be a valid option here as well?

https://github.com/libp2p/rust-libp2p/blob/master/wasm-tests/webtransport-tests/README.md

Reply:
Yes for sure, I can minimize the example for the browser and eliminate the framework

Reply:
Thank you @DougAnderson444